tests/feature_creation_tests.py

```diff
@@ -1,9 +1,15 @@
 import pandas as pd
-
+import os
+import sys
+sys.path.append(os.path.abspath(os.path.join(os.path.dirname(__file__), '../')))
+print(sys.path)
+from features.taped_stats import TapedStats
 class FeatureCreationTests():
     def __init__(self) -> None:
         self.fights_df = pd.read_csv('tests/test_fights.csv', encoding='latin-1')
         self.fighter_df = pd.read_csv('tests/test_fighters.csv', encoding='latin-1')
+        self.taped_stats = TapedStats()
```

Comment:
Could you move this up or below a couple of rows so that our feature objects and our dfs are separated pls